docs/marionette.regionmanager.md

```diff
@@ -103,7 +103,7 @@ var rm = new Marionette.RegionManager();
 var regions = rm.addRegions({
   main: '#main-content',
   navigation: {
-    selector: '#navigation',
```

Comment:
Excellent, I'll chop it out. I'll also put a deprecated notice at the top of `marionette.regionmanager.md` directing readers to `marionette.region.md`